extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -286,10 +286,15 @@ export class AmpConsent extends AMP.BaseElement {
       let isConsentRequiredPromise;
 
       if (this.consentConfig_[instanceId]['checkConsentHref']) {
-        isConsentRequiredPromise = this.getConsentRemote_(instanceId).then(
-            response => {
-              this.parseConsentResponse_(instanceId, response);
-            });
+        const remoteConsentPromise = this.getConsentRemote_(instanceId);
+        isConsentRequiredPromise = remoteConsentPromise.then(response => {
+          return this.parseConsentResponse_(instanceId, response);
+        });
+        const sharedDataPromise = remoteConsentPromise.then(response => {
+          return response ? response['sharedData'] : null;
```

Comment:
Looks like the promise can return `undefined` here.

Reply:
should be fine right? it can be `undefined` or `null`.

Reply:
done